processor/src/main/resources/org/mapstruct/ap/internal/model/Setter.ftl

```diff
@@ -0,0 +1,10 @@
+ <#--
+
+    Copyright MapStruct Authors.
+
+    Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+public void ${name}( <@includeModel object=parameter/> ) {
+    this.<@includeModel object=field/> = <@includeModel object=parameter/>;
+}
```

Comment:
I'm unable to make this mapping work. The result looks like this:

```java
    public void setGenderSpringSetterMapper(  ) {
        this. = ;
    }
```

Reply:
@filiphr this should be the last big thing that needs to be fixed but for the life of me I'm not able to figure out how this is supposed to work. Some help would be greatly appreciated